index.js

```diff
@@ -33,26 +33,30 @@ function runReadmeGenerator(options) {
   const valuesFilePath = options.values;
   const readmeFilePath = options.readme;
   const schemaFilePath = options.schema;
+  const versionFlag = options.version;
 
-  if (!readmeFilePath && !schemaFilePath) {
-    throw new Error('Nothing to do. Please provide a README file or Schema file output.');
-  }
-  if (!valuesFilePath) {
-    throw new Error('Values file not provided');
-  }
-
-  const configPath = options.config ? options.config : `${__dirname}/config.json`;
-  const CONFIG = require(configPath);
-  const parametersList = getParameters(options);
+  if (versionFlag) {
+    console.log("Version:", require('./package.json').version);
+  } else {
+    if (!readmeFilePath && !schemaFilePath) {
+      throw new Error('Nothing to do. Please provide a README file or Schema file output.');
```

Comment:
It could be helpful to be explicit about the options, e.g.:

```suggestion
      throw new Error('Nothing to do. You must provide the --readme or --schema options.');
```